src/commands/cmd_constraint.c

```diff
@@ -366,16 +369,16 @@ static bool _Constraint_Create
 	return res;
 }
 
-// command handler for GRAPH.CONSTRAIN command
-// GRAPH.CONSTRAIN <key> CREATE UNIQUE/MANDATORY [NODE label / RELATIONSHIP type] PROPERTIES prop_count prop0, prop1...
-// GRAPH.CONSTRAIN <key> DROP   UNIQUE/MANDATORY [NODE label / RELATIONSHIP type] PROPERTIES prop_count prop0, prop1...
+// command handler for GRAPH.CONSTRAINT command
+// GRAPH.CONSTRAINT CREATE <key> UNIQUE/MANDATORY [NODE label / RELATIONSHIP type] PROPERTIES prop_count prop0, prop1...
+// GRAPH.CONSTRAINT DROP <key> UNIQUE/MANDATORY [NODE label / RELATIONSHIP type] PROPERTIES prop_count prop0, prop1...
 int Graph_Constraint
 (
 	RedisModuleCtx *ctx,
 	RedisModuleString **argv,
 	int argc
 ) {
-	if(argc < 9) {
+	if(argc < 8) {
```

Comment:
1. GRAPH.CONSTRAINT
2. DROP
3. key
4. UNIQUE/MANDATORY
5. NODE
6. label
7. PROPERTIES
8. prop_count
9. prop0

Reply:
I know. but @LiorKogan asked, me to return a different error when prop0 is 0:
See the first request.

Reply:
I think we should return an arity error in the example you've provided and 
`ERR wrong number of arguments for 'graph.constraint' command` when the user specified something like:
`graph.constraint g CREATE MANDATORY NODE label PROPERTIES -1 age`

Reply:
Generally, I'm assuming that the user is confused / unsure about the syntax / experimenting, and I'm trying to reply with the most informative error message that can help them understand their mistake. That was the motivation here.

Reply:
I think the Arity error should turn the user to the documentation

Reply:
From the code perspective I think I like better the Arity, but from user perspective I like the error msg suggested by @LiorKogan

Reply:
ok.